mbuild/formats/xyz.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+
+import mbuild as mb
+
+__all__ = ['read_xyz']
+
+
+def read_xyz(filename):
+    """Read an XYZ file.
```

Comment:
I think it might be a good idea to provide a brief description of the expected file format here, since there are no official specifications for XYZ files (e.g. the [TINKER XYZ format](http://chembytes.wikidot.com/tnk-tut00#toc2) would cause problems with our reader).